governance/polls/MIP9 Community Greenlight Poll - RAI - September 06, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - RAI (Rai Reflex Index) - September 06, 2021
```

Comment:
We haven't tended to do leading 0s for dates previously. Super minor point but eh, consistency.